### PR TITLE
add hello-bar

### PR DESCRIPTION
**Pull request by jdorfman**
image

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -79,3 +79,30 @@
...
+    background: "#000",
+    size: "large",
+    duration: 1000,
+    align: "center",
+    targeting: {
+      once: false,
+    },
+  });
+
+  // window.ga("create", "G-GS3KLXMBN3", "auto");
```
Comment (jdorfman) [inline _includes/footer.html:105]: remove

File: _includes/footer.html [hunk 2]
```diff
@@ -79,3 +79,30 @@
...
+  // helloBar.on("cta-mouse-click", e => window.ga("send", "event", "ab-test", e.getAttribute("href")));
```
Comment (jdorfman) [inline _includes/footer.html:106]: remove

File: _includes/footer.html [hunk 3]
```diff
@@ -79,3 +79,30 @@
 
 <script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.5.1.min.dc5e7f18c8.js?site=5f906e60f009d620eb2024dd" type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
 <script defer async src="/js/curiefense.js" type="text/javascript"></script>
+
+
+<script src="https://cdn.jsdelivr.net/npm/hello-bar@1.4.0/build/index.js"></script>
```
Comment (jdorfman) [inline _includes/footer.html:84]: add `defer async`

File: _layouts/default.html [hunk 4]
```diff
@@ -76,6 +76,7 @@
         background-image: url(/images/tick_contrast.png);
       }
     </style>
+    <script src="https://cdn.jsdelivr.net/npm/hello-bar@1.4.0/build/index.js"></script>
```
Comment (jdorfman) [inline _layouts/default.html:79]: duplicate, remove this one.

File: _includes/footer.html (hunk 3 above)
Comment (Waleed-Iqbal) [reply _includes/footer.html:84]: We don't need that
we have to make sure that the `hello-bar` is loaded before we do `const helloBar = new HelloBar['default']` in the next script

**Review by jdorfman — COMMENTED** [review]
Test to make sure this works:
```
targeting: {
      once: true,
```

**Comment by jdorfman** [comment]
The `X` isn't aligned center vertical 

image

**Pull request merged**